proto/desmos/subspaces/v1/query.proto

```diff
@@ -0,0 +1,150 @@
+syntax = "proto3";
+package desmos.subspaces.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "desmos/subspaces/v1/models.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v2/x/subspaces/types";
+
+// Query defines the gRPC querier service
+service Query {
+
+  // Subspaces queries all the subspaces inside Desmos
+  rpc Subspaces(QuerySubspacesRequest) returns (QuerySubspacesResponse) {
+    option (google.api.http).get = "/desmos/subspaces/v1/subspaces";
+  }
+
+  // Subspace queries all the information about the subspace with the given id
+  rpc Subspace(QuerySubspaceRequest) returns (QuerySubspaceResponse) {
+    option (google.api.http).get =
+        "/desmos/subspaces/v1/subspaces/{subspace_id}";
+  }
+
+  // UserGroups queries all the groups that are present inside the subspace with
+  // the given id
+  rpc UserGroups(QueryUserGroupsRequest) returns (QueryUserGroupsResponse) {
+    option (google.api.http).get = "/desmos/subspaces/v1/groups/{subspace_id}";
+  }
+
+  // UserGroup queries the user group having the given id inside the specific
+  // subspace
+  rpc UserGroup(QueryUserGroupRequest) returns (QueryUserGroupResponse) {
+    option (google.api.http).get =
+        "/desmos/subspaces/v1/groups/{subspace_id}/{group_id}";
+  }
+
+  // UserGroupMembers queries all the members of a given user group
+  rpc UserGroupMembers(QueryUserGroupMembersRequest)
+      returns (QueryUserGroupMembersResponse) {
+    option (google.api.http).get =
+        "/desmos/subspaces/v1/groups/{subspace_id}/{group_id}/members";
+  }
+
+  // UserPermissions queries the permissions for the given user
```

Comment:
`UserPermissions` query is for the specific user. I think Having `GroupPermissions` query would be helpful.

Reply:
Never mind I found it have been included in the `UserGroup` method already.